macos/docs/Controls/Button.md

```diff
@@ -0,0 +1,36 @@
+# Button
+
+## Overview
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+![ButtonViews.png](.attachments/ButtonViews.png)
+
+## Best Practices
+### Do
+- Make sure the label conveys a clear purpose of the button to the user.
+- Describe the action the button will perform, ideally with a verb. Use concise, specific, self-explanatory labels, usually a single word.
+
+### Don't
+- Place the default focus on a button that performs a destructive actionn. Instead, place the default focus on the button that performs the "safe action" (e.g "Cancel").
+
+## Usage
+```Swift
+// There are 3 styles to choose from, the default being primary filled
+let primaryFilledButton = Button(title: "FluentUI Button", style: .primaryFilled),
```

Comment:
Do we describe when each should be used?

Reply:
That would be great :D, if I knew when each was actually used.
Context: I chose these 3 styles by looking at what we do on other platforms, and the documentation for each platform didn't specify when to use each one. I'm not sure that is standardized.

Reply:
Sounds like we can file an issue on PM/Design for missing documentation?

Reply:
Perhaps. Considering I just "engineering best guessed" the design and then validated it with design in post, I feel once we have clients use the button, we'll get more design tokens / redlines / specifc requests, and we can update documention to match those new things. Regardless, I'll try and bring up this area is lacking